src/applications/vaos/tests/new-appointment/components/DateTimeSelectPage/helpers.js

```diff
@@ -0,0 +1,99 @@
+import moment from 'moment';
+
+import {
+  mockAppointmentSlotFetch,
+  mockEligibilityFetches,
+} from '../../../mocks/helpers';
+import { getAppointmentSlotMock, getClinicMock } from '../../../mocks/v0';
+
+export function setDateTimeSelectMockFetches({
```

Comment:
I like this idea in general. I think we'll want to come up with some guidelines to follow, since I think we want to balance hiding common mock data setup with keeping data that's important for a particular test in that test.

We should definitely hide as much as we can as far as the set up on prior pages, and I like having booleans to toggle between different scenarios we might need.

I'm a little more wary of completely hiding the slot dates in a mock function, because that's info that we're going to use in the test, it looks like we're picking magic times if you haven't dug into the mock function. In this case, I wonder if we could have like a slotDates array that takes in slot dates and a slotDatesByClinic object, which lets you do the more complicated setting up of one slot date for different clinics?

Reply:
The method currently has a `slotsToSet` array that I'm using to set slots for any tests that need to customize the slots needed. The tests for custom dates, and moving from month to month use that param. Is that different from what you're describing?

Reply:
I was looking at the singleSlot and singleClinic params and wondering if we could derive those based on the slots passed in slotsToSet, basically.

Reply:
Ok got it. I moved all of the slot time data out of the function and added a `slotDatesByClinicId` object param.